docs/pages/guides/typescript.mdx

```diff
@@ -4,51 +4,62 @@ description: An in-depth guide on configuring an Expo project with TypeScript.
 ---
 
 import { Terminal } from '~/ui/components/Snippet';
-
-> Example project: [with-typescript](https://github.com/expo/examples/tree/master/with-typescript)
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+import { Tabs, Tab, TabsGroup } from '~/ui/components/Tabs';
 
 Expo has first-class support for [TypeScript](https://www.typescriptlang.org/). The JavaScript interface of the Expo SDK is completely written in TypeScript.
 
-To get started, create a **tsconfig.json** in your project root:
+<BoxLink
+  title="with-typescript"
+  description="See the example project on GitHub."
+  href="https://github.com/expo/examples/tree/master/with-typescript"
+  Icon={GithubIcon}
+/>
+
+<TabsGroup>
+
+## Get started
+
+### Quick start with a template
+
+The easiest way to get started is to initialize your new project using a TypeScript template:
+
+<Terminal cmd={['$ npx create-expo-app -t expo-template-blank-typescript']} />
+
+Then, to type-check the project, run the following command:
 
-<Terminal cmd={['$ touch tsconfig.json']} />
+<Tabs>
+<Tab label="npx">
 
-For SDK 48 and above, running `npx expo start` will prompt you to install the required dependencies, such as `typescript` and `@types/react`, and automatically configure your **tsconfig.json**. For SDK 47 and below, the command will also prompt to install `@types/react-native` as an additional dependency.
+<Terminal cmd={['$ npx tsc']} />
```

Comment:
Hmm, stand-alone `tsc` is deprecated for a while - https://www.npmjs.com/package/tsc - the correct alternative is to call `yarn/npm tsc --noEmit`, with the TypeScript package installed first, because running TS via exec (`npx`) is disabled.

Reply:
Also, adding `"tsc": "tsc"` as a script in `package.json` works in case of npm:

<img width="654" alt="CleanShot 2023-07-10 at 18 06 09@2x" src="https://github.com/expo/expo/assets/10234615/cbbdc32c-bc13-4169-9fc7-1383f9b6c995">

<img width="274" alt="CleanShot 2023-07-10 at 18 06 23@2x" src="https://github.com/expo/expo/assets/10234615/55900680-5fa8-4c9c-8c30-ce73ab264a52">

<img width="333" alt="CleanShot 2023-07-10 at 18 06 32@2x" src="https://github.com/expo/expo/assets/10234615/2758c982-1107-488f-a62c-8bb345a01194">

So I think we can recommend adding `tsc` as package script?

Reply:
@Simek, added the  tsc script instructions.

<img width="922" alt="CleanShot 2023-07-11 at 16 18 52@2x" src="https://github.com/expo/expo/assets/10234615/76c3529c-3d65-4105-a2ee-80c81c2ab481">